starter-code/experiment/index.js

```diff
@@ -0,0 +1,57 @@
+// const mongoose = require("mongoose");

```

Comment:
you might want to take out this code from here since it's not relevant for the project drones...